app/views/_create-secret.html

```diff
@@ -0,0 +1,267 @@
+      <div>
+        <input class="form-control"
+          type="email"
+          id="passwordToken"
+          name="passwordToken"
+          ng-model="newSecret.data.dockerMail"
+          type="password"
+          autocorrect="off"
+          autocapitalize="off"
+          spellcheck="false"
+          required>
+      </div>
+    </div>
+  </div>
+
+  <div ng-if="newSecret.authType === 'Docker Config'">
+    <div class="form-group" id="docker-config">
+      <label for="dockerConfig" class="required">Docker Configuration File</label>
+      <osc-file-input 
+        model="newSecret.data.dockerConfig"
+        drop-zone-id="docker-config"
+        dragging="false"
+        help-text="Upload .dockercfg or config.json file"
+        ng-disabled="false"
+        show-values="false"></osc-file-input>
+      <div ui-ace="{
+        mode: 'json',
+        theme: 'eclipse',
+        onLoad: aceLoaded,
+        rendererOptions: {
+          fadeFoldWidgets: true,
+          showPrintMargin: false 
+        }
+      }" ng-model="newSecret.data.dockerConfig" class="create-secret-editor ace-bordered yaml-mode" id="dockerconfig-editor" required></div>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <div class="checkbox">
+      <label>
+        <input type="checkbox" ng-model="newSecret.linkSecret"/>
+        Link with secret with Service Account
+        <span class="help action-inline">
+          <a href>
+            <i class="pficon pficon-help" aria-hidden="true"
+            data-toggle="tooltip" data-original-title="Linking tooltip">
+          </i>
+          </a>
+        </span>
+      </label>
+    </div>
+  </div>
+
+  <div ng-if="newSecret.linkSecret">
+    <div class="form-group">
+      <label class="required">Service Account</label>
+      <ui-select required ng-model="newSecret.serviceAccountToLink">
+        <ui-select-match placeholder="Service Account Name">{{$select.selected}}</ui-select-match>
+        <ui-select-choices repeat="sa in (serviceAccountsNames | filter : $select.search)">
+          <div ng-bind-html="sa | highlight : $select.search"></div>
+        </ui-select-choices>
+      </ui-select>
+    </div>
+
+    <div class="form-group" >
+      <label class="required">Link as</label><br/>
+
+      <div class="form-group">
+        <div class="checkbox">
+          <label>
+            <input type="checkbox" ng-model="newSecret.linkAs.secrets" ng-checked="newSecret.type !== 'Pull Image Secret'"/>
+            Link as a secret that will be used by pods running using picked service account.
+          </label>
+        </div>
+      </div>
+
+      <div class="form-group">
+        <div class="checkbox">
+          <label>
+            <input type="checkbox" ng-model="newSecret.linkAs.imagePullSecrets" ng-checked="newSecret.type === 'Pull Image Secret'"/>
+            Link as a image pull secret that will be used for pulling any images in pods that reference picked service account.
```

Comment:
Maybe "Assign as an image pull secret for pods that use the service account."

@jwforres word smithing help? :)